components/actiongroup/package.json

```diff
@@ -14,20 +14,14 @@
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
   "main": "dist/index-vars.css",
-  "scripts": {
-    "build": "gulp",
-    "clean": "rimraf dist"
-  },
   "peerDependencies": {
     "@spectrum-css/actionbutton": ">=5",
     "@spectrum-css/tokens": ">=13"
   },
-  "devDependencies": {
-    "@spectrum-css/actionbutton": "^5.0.16",
-    "@spectrum-css/component-builder-simple": "^3.0.1",
-    "@spectrum-css/icon": "^4.0.5",
```

Comment:
None of our packages need `devDependencies` anymore because the build work happens at root now. The only reason we had them previously was for storybook but storybook doesn't need them either in a monorepo environment.

Reply:
Oh ok, and since this was never a peer dependency it doesn't need to be one now 👍